data/hypervisor.args.in

```diff
@@ -2,7 +2,7 @@
 -name
 @NAME@
 -machine
-pc-lite,accel=kvm,kernel_irqchip,nvdimm
+pc,accel=kvm,kernel_irqchip,nvdimm
```

Comment:
@egernst we should call this out somewhere in the documentation that we are currently using pc vs pc-lite.